src/core/frame.h

```diff
@@ -163,7 +165,8 @@ CXPLAT_STATIC_ASSERT(
 #define QUIC_FRAME_IS_KNOWN(X) \
```

Comment:
Also update `QuicConnRecvFrames` to process the frames. Even it's a no-op for now.

Reply:
Added case.